build(docs-infra): update `.browserslistrc` to align with new CLI apps

gkalpak opened the pull request:
Update `.browserslistrc` to align with new CLI apps. This drops some older browsers that are no longer supported by Angular v13+. See the [list of supported browsers][1].

##
~~Blocked on #44285.~~

[1]: https://angular.io/guide/browser-support

mary-poppins commented:
You can preview 79dd36b at https://pr44283-79dd36b.ngbuilds.io/.

mary-poppins commented:
You can preview 5b0b9f9 at https://pr44283-5b0b9f9.ngbuilds.io/.

alan-agius4 reviewed (approved):
Thanks for this @gkalpak

jessicajaniuk reviewed (approved):
LGTM :cookie:

reviewed-for: size-tracking

dylhunn reviewed (approved):
LGTM!

dylhunn commented:
LGTM
reviewed-for: size-tracking

dylhunn reviewed (approved):
reviewed-for: size-tracking

mary-poppins commented:
You can preview 4e3c049 at https://pr44283-4e3c049.ngbuilds.io/.

gkalpak commented:
FYI: Rebased on master to resolved conflicts.

mary-poppins commented:
You can preview da0c013 at https://pr44283-da0c013.ngbuilds.io/.

dylhunn commented:
This PR has merge conflicts with 13.0.x patch branch. Mind rebasing it?

gkalpak committed:
build(docs-infra): update `.browserslistrc` to align with new CLI apps

Update `.browserslistrc` to align with new CLI apps. This drops some
older browsers that are no longer supported by Angular v13+.
See the [list of supported browsers][1].

[1]: https://angular.io/guide/browser-support

mary-poppins commented:
You can preview 1d05524 at https://pr44283-1d05524.ngbuilds.io/.

mary-poppins commented:
You can preview 1545a34 at https://pr44283-1545a34.ngbuilds.io/.

gkalpak commented:
@dylhunn, this is because master and `13.0.x` have different numbers in `aio-payloads.json` (so not something that can be fixed with rebasing) 🤷‍♂️
Let's merge this to master only and I have created a separate PR to target `13.0.x` specifically: #44352

mary-poppins commented:
You can preview 4d8ca7d at https://pr44283-4d8ca7d.ngbuilds.io/.

mary-poppins commented:
You can preview d17456d at https://pr44283-d17456d.ngbuilds.io/.
